kivymd/uix/datatables/datatables.py

```diff
@@ -326,23 +334,32 @@ def __init__(self, **kwargs):
                     (
                         CellHeader(
                             text=col_heading[0],
-                            sort_action=col_heading[2],
+                            tooltip=col_heading[2],
+                            sort_action=col_heading[3],
                             width=self.cols_minimum[i],
                             table_data=self.table_data,
                             is_sorted=(col_heading[0] == self.sorted_on),
                             sorted_order=self.sorted_order,
                         )
-                        if len(col_heading) == 3
+                        if len(col_heading) == 4
                         else CellHeader(
                             text=col_heading[0],
+                            tooltip=col_heading[2],
                             width=self.cols_minimum[i],
                             table_data=self.table_data,
                         )
+                        if len(col_heading) == 3
+                        else CellHandler(
+                            text=col_heading[0],
+                            width=self.cols_minimum[i]
+                            table_data=self.table_data,
+                        )
```

Comment:
![Снимок экрана 2022-09-12 в 15 53 43](https://user-images.githubusercontent.com/16930280/189658958-7c249fe2-1003-48e0-bbb2-552c3f0d0974.png)

Reply:
Oh yeah, sorry about that. I had made changes in my version of kivymd and was trying to just hand type them into my contribution fork to ready them for a PR, guess I missed to throughly check before submitting. Let me fix that in case you haven't fixed it.